src/Types/index.d.ts

```diff
@@ -37,6 +37,7 @@ declare interface Flight {
   duration: FlightDuration;
   departureTime: string;
   arrivalTime: string;
+  cost: number;
```

Comment:
@johnkahn Thoughts on making this a string? We'd be able to consistently keep it at two decimals and prevent people from needing to truncate on their own when they retrieve data.

Reply:
I'm good with leaving this as a number, but @johnkahn any final thoughts before we wrap this up?

Reply:
It should be a number since this is the API. The UI should be in charge of formatting (`.toFixed(2)` in javascript)